setup_helpers.py

```diff
@@ -84,9 +85,16 @@ def read_metadata(config_filename):
 
 
 def read_readme(readme_filename):
-    with open(readme_filename) as ff:
+    with codecs.open(readme_filename, encoding='utf8') as ff:
         lines = ff.read().splitlines()
 
+    # Skip lines that contain raw HTML markup
+    lines = lines[:4] + lines[26:]
```

Comment:
This seems a bit dangerous - what if someone else makes the README and doesn't realize they need to update this file?

Maybe instead look for the `.. raw:: html` marker, and then skip all consecutive following lines that are indented (or empty)?

Reply:
Yes, you're absolutely right. We should do something smarter like that in the long term but for the sake of this bug fix I'm going to leave it alone.

Reply:
Also the point of the `twine check` above is that it should catch any such changes that lead to invalid output, but we're in agreement.